cli/template/extras/src/pages/index/with-auth-trpc-tw.tsx

```diff
@@ -74,7 +74,7 @@ const AuthShowcase: React.FC = () => {
       </p>
       <button
         className="rounded-full bg-white/10 px-10 py-3 font-semibold text-white no-underline transition hover:bg-white/20"
-        onClick={sessionData ? () => signOut() : () => signIn()}
+        onClick={sessionData ? () => void signOut() : () => void signIn()}
```

Comment:
fixes @typescript-eslint/no-missused-promises

Reply:
Are you sure it's ok for those promises to be floating? Is it possible for them to reject in a way the app should handle?

Reply:
pretty sure this is fine - I usually do this.

I think the only scenario which you shouldn't do this is if you're using credentials